catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/QualifiedMetacardTypeImpl.java

```diff
@@ -41,10 +43,10 @@ public QualifiedMetacardTypeImpl(String namespace, String name,
             Set<AttributeDescriptor> descriptors) {
         super(name, descriptors);
 
-        this.namespace = namespace;
-
-        if (namespace == null || namespace.isEmpty()) {
+        if (StringUtils.isBlank(namespace)) {
```

Comment:
`this.namespace = StringUtils.defaultIfBlank(namespace, QualifiedMetacardType.DEFAULT_METACARD_TYPE_NAMSPACE)`?